pkg/kubelet/dockershim/docker_service.go

```diff
@@ -311,6 +319,89 @@ type dockerService struct {
 
 // TODO: handle context.
 
+// GetRuntimeConfigInfo returns the runtime config.
+func (ds *dockerService) GetRuntimeConfigInfo(_ context.Context, r *runtimeapi.GetRuntimeConfigInfoRequest) (*runtimeapi.GetRuntimeConfigInfoResponse, error) {
+	dockerInfo, err := ds.client.Info()
+	glog.Infof("Docker Info: %+v", dockerInfo)
+	if err != nil {
+		glog.Errorf("Failed to execute Info() call to the Docker client: %v", err)
+		return nil, err
+	}
+	uidMapping := &runtimeapi.LinuxIDMapping{ContainerId: uint32(0)}
```

Comment:
can we fetch this information once on startup of the kubelet similar to how we collect cgroup driver information?

in this way, we can use it in `NewDockerService`.

recomputing it on every call seems unnecessary.